theme/scripts/components/Component.ts

```diff
@@ -26,6 +26,23 @@ export default class Component<D> {
 		this.init()
 	}
 
+	protected getChild<C extends HTMLElement>(
+		selector: string,
+		F: Constructor<C>,
```

Comment:
default value to `HTMLElement` is not wanted?

Reply:
Cc. @JirkaVebr